fdbserver/workloads/MetaclusterManagementWorkload.actor.cpp

```diff
@@ -771,7 +824,10 @@ struct MetaclusterManagementWorkload : TestWorkload {
 				ASSERT(exists);
 				return Void();
 			} else if (e.code() == error_code_metacluster_no_capacity) {
-				ASSERT(!hasCapacity && !exists);
+				ASSERT(!exists && !hasCapacity);
+				// It's possible that all the data clusters are excluded from the auto-assignment pool.
```

Comment:
This comment might be clearer up above where you set `hasCapacity`

Reply:
Done